src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidatorTest.java

```diff
@@ -772,7 +864,7 @@ void testErrorReportedForMissingUpdateEntityNameField() {
         assertError(qualifiedFieldName, validationMessage, errors);
     }
 
-    private void testFullUpdateRemoveValidationWithoutTrusts() {
+    private Errors testFullUpdateRemoveValidationWithoutTrusts() {
```

Comment:
Why did this method signature have to change and the `assertFalse(errors.hasErrors());` line move out of the method?

Same question applies to the methods below.